api/docs_handler.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/topfreegames/pitaya"
+
+	"github.com/topfreegames/pitaya-admin/usecases"
+
+	"github.com/topfreegames/pitaya-admin/constants"
+)
+
+// DocsHandler handle docs
+type DocsHandler struct {
+	App             *App
+	docsRemoteRoute string
+}
+
+// NewDocsHandler create new docs handler
+func NewDocsHandler(a *App) *DocsHandler {
+	m := &DocsHandler{
+		App:             a,
+		docsRemoteRoute: a.Config.GetString("pitayaadmin.docs.suffix"),
+	}
+	return m
+}
+
+// ServeHTTP method. Docs will only work if game has a remote for docs
+func (s *DocsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	v := r.URL.Query()
+	param := v.Get("type")
+	roh := v.Get("t")
```

Comment:
I'm not sure I understand what `roh` means, please give the variables more explicit names. Also, `t` is not clear either.